src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.figure.enums.Color;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, Color color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public double getArea() {
```

Comment:
create interface `AreaCalculator` with method `getArea()`

```suggestion
    @Override
    public double getArea() {
```